futurex_openedx_extensions/helpers/models.py

```diff
@@ -34,6 +38,123 @@ class Meta:
         unique_together = ('view_name', 'allowed_role')
 
 
+class ViewUserMappingManager(models.Manager):  # pylint: disable=too-few-public-methods
```

Comment:
this is needed to allow smooth UX in django-admin when sorting and filtering on column values